gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanJobLauncher.java

```diff
@@ -49,7 +58,7 @@
  *
  * @author Yinan Li
  */
-public class AzkabanJobLauncher extends AbstractJob {
+public class AzkabanJobLauncher extends AbstractJob implements ApplicationLauncher, JobLauncher {
```

Comment:
It may be a little confusing. Because `job.launcher.type` can only be LOCAL, MAPREDUCE or YARN. It cannot be AzkabanJobLauncher. So technically it's not a job launcher, at least not in the same sense as LocalJobLauncher or MRJobLauncher.